tests/test_admin.py

```diff
@@ -92,3 +93,69 @@ def test_admin(app):
             data={'rowid': -42, 'action': 'activate'},
             follow_redirects=True
         )
+
+    user_adminview['model'] = user_model
```

Comment:
Maybe line 55 and 56 should use `get` instead of `pop`. (`user_model = user_adminview.get('model')`)